### DISCUSSION TITLE
Hide menu but be able to get into 'edit dashboard'

### DISCUSSION THREAD

**Discussion by Patrick1610** [NemesisRE/kiosk-mode#115] [Q&A]
Hi, I want to use this integration to hide the menu for all users, including myself. Although, I would like to still be able to use my phone to edit a dashboard on the go. Currently, I don't see an option where I can hide the top header but still have acces to the ability to edit the dashboard. 

Is there any way around this? 
Or can I have a button inside Home Assistant to enable and disable the header with the flick of a switch? 

Kind regards, Patrick

**Comment by elchininet** [comment]
Hi @Patrick1610,

>Or can I have a button inside Home Assistant to enable and disable the header with the flick of a switch?

For that you can use [entity_settings](https://github.com/NemesisRE/kiosk-mode#entity_settings).

![kiosk_entity_settings](https://github.com/NemesisRE/kiosk-mode/assets/3728220/0a55be2d-f067-4361-97b5-29bc3cb354cc)
